docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -28,7 +28,9 @@ any of the following:
 - a [Linux VM running on the Windows development machine](HyperVLinuxVMSetup.md)
 
 Ideally, the machine should be SGX capable (see [instructions for determining the SGX support level](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) if needed),
-but a non-SGX machine can still be used in simulation mode.
+but a non-SGX machine can still be used:
+ - in simulation mode, or
+ - if you would like to install the Host-Verify SDK and build the `host-verify` sample.
```

Comment:
"if you install the Host-Verify SDK and build evidence verification applications without enclave"